### PR TITLE
fix(VCard): fix word wrapping & color

### PR DESCRIPTION
**Pull request by avengerweb**
## Motivation and Context
Fixes:
#7920 
#7730 

## How Has This Been Tested?
visually

## Markup:
VCard color and long title:
<details>

```vue
<template>
  <v-container style="width: 250px">
    <v-card dark>
      <v-card-title primary-title>
          <h3 class="headline mb-0">Kangaroo Valley Safari In Australia Supercalifragilisticexpialidocious</h3>
      </v-card-title>
      <v-card-text>
        <v-card light>
          <v-card-text>Text with right color</v-card-text>
        </v-card>
      </v-card-text>
    </v-card>
  </v-container>
</template>

<script>
export default {
  data: () => ({
    //
  })
}
</script>

```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.

### CODE REVIEW

**Comment by sh7dm** [comment]
Hey @avengerweb you probably need to separate card and dialog into different prs. They look unrelated to each other.

**Comment by avengerweb** [comment]
@sh7dm done

File: packages/vuetify/src/components/VCard/VCard.sass [hunk 1]
```diff
@@ -17,6 +17,8 @@
   outline: none
   text-decoration: none
   transition-property: box-shadow, opacity
+  word-break: break-word
```
Comment (KaelWD) [inline packages/vuetify/src/components/VCard/VCard.sass]: word-break is deprecated

Comment (avengerweb) [reply packages/vuetify/src/components/VCard/VCard.sass]: IE don’t think so,
https://github.com/vuetifyjs/vuetify/blob/5c61eee424f982cf2d8c9c8b9f411552101a6749/packages/vuetify/src/styles/settings/_utilities.scss#L322

Comment (KaelWD) [reply packages/vuetify/src/components/VCard/VCard.sass]: Maybe that needs updating too then haha, MDN says it's not supported at all in IE/edge and deprecated in other browsers.

Comment (avengerweb) [reply packages/vuetify/src/components/VCard/VCard.sass]: It works in IE 11 with that rule, and not working without...

**Pull request merged**